packages/cfd/src/Containers/jurisdiction-modal/jurisdiction-modal-content-items/jurisdiction-modal-checkbox.tsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import { Text, Checkbox, StaticUrl } from '@deriv/components';
+import { isMobile, getAuthenticationStatusInfo } from '@deriv/shared';
+import { Localize } from '@deriv/translations';
+import { TJurisdictionCheckBoxProps } from '../../props.types';
+
+const JurisdictionCheckBox = ({
+    account_status,
+    class_name,
+    is_checked,
+    jurisdiction_selected_shortcode,
+    onCheck,
+}: TJurisdictionCheckBoxProps) => {
+    const { poi_poa_verified_for_bvi_labuan_maltainvest, poi_poa_verified_for_vanuatu } =
+        getAuthenticationStatusInfo(account_status);
+
+    const showCheckBox = () => {
+        if (jurisdiction_selected_shortcode) {
+            if (jurisdiction_selected_shortcode === 'svg') {
+                return false;
+            } else if (jurisdiction_selected_shortcode === 'vanuatu' && poi_poa_verified_for_vanuatu) {
+                return true;
+            } else if (
+                (jurisdiction_selected_shortcode === 'bvi' ||
+                    jurisdiction_selected_shortcode === 'labuan' ||
+                    jurisdiction_selected_shortcode === 'maltainvest') &&
+                poi_poa_verified_for_bvi_labuan_maltainvest
```

Comment:
Less repetition. Refer to the `OR/Array.includes` comment as per above
```suggestion
                (['bvi', 'labuan', 'maltainvest'].includes(jurisdiction_selected_shortcode)) &&
                poi_poa_verified_for_bvi_labuan_maltainvest
```